pkg/karmadactl/cmdinit/utils/util_test.go

```diff
@@ -79,3 +80,42 @@ func TestDownloadFile(t *testing.T) {
 		t.Errorf("DeCompress() got %v, want %v", got, want)
 	}
 }
+
+func TestListFiles(t *testing.T) {
+	tests := []struct {
+		name     string
+		path     string
+		tempfils []string
+	}{
+		{
+			name:     "get files from path",
+			path:     "temp-path" + randString(),
+			tempfils: []string{"tempfiles1" + randString(), "tempfiles2" + randString()},
+		},
+		{
+			name:     "no files from path",
+			path:     "temp-path" + randString(),
+			tempfils: []string{},
+		},
+	}
+	for _, tt := range tests {
+		err := os.Mkdir(tt.path, 0755)
+		if err != nil {
```

Comment:
add `defer os.RemoveAll(tt.path)` here, or if `t.Fatal`, the directory will not be removed.